src/libs/utils.js

```diff
@@ -186,10 +186,12 @@ export const generateClusterName = () => `saturn-${uuid()}`
 
 export const waitOneTick = () => new Promise(setImmediate)
 
-export const entityAttributeText = value => {
+export const entityAttributeText = (value, machineReadable) => {
   return cond(
     [_.has('entityName', value), () => value.entityName],
-    [_.has('items', value), () => `${value.items.length} items`],
+    [_.has('items', value), () => machineReadable ?
+      JSON.stringify(value.items) :
```

Comment:
TOL: This does bring us in line with what the server produces in this case, but I suspect that behavior might not be working as intended. If you download and re-upload this, you get a string attribute with the JSON contents, not the original array. There's a similar issue with the `entityName` case above. If the goal is to preserve the data across a download/upload, we might need to fix the server (and also change this code).

Reply:
Checked with Sushma, apparently this is a known thing. Leaving it as-is for now.